scripts/install/nltk_dictionaries.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+set -eo pipefail
+
+echo "MACHINE LEARNING WORKER NOTE: Next, you will be asked to install some wordlists for the natural language processing parts of Augur.  Please be advised that the machine learning workers, including the clustering worker, discourse worker, messsage_analysis_worker, pull_request_analysis_worker, and the LSTM model of the insight_worker will not function fully without these lists."
```

Comment:
This line is a duplicate of line 45 of `scripts/install/install.sh`. Please remove one or the other.